### DISCUSSION TITLE
MBTI簡介

### DISCUSSION THREAD

**Discussion by JessyCheng2003** [1101-GS4719-C/Python-Community-GS4719-C#1396] [General]
### 簡介
MBTI 又稱為「邁爾斯-布理格斯性格分類指標」（Myers-Briggs Type Indicator），是一種性格分類模式，是美國作家 Katherine Cook Briggs 與女兒 Isabel Briggs Myers 因為對心理學的愛好，以心理學家榮格在 1921 年的《心理類型》為基礎，深入研究後加以提出。旨在幫助人們了解自己的人格類型和與他人互動的方式。

### 評估方式

MBTI測試通過問卷調查來評估參與者的偏好和行為模式。將人格分為四個二分類別：外向或內向（E或I）、感性或直覺（S或N）、思考或感受（T或F）以及判斷或知覺（J或P）。人們根據這四種偏好類型被歸類為16種不同的人格類型。

E：外向（Extraversion），外向的人較注重社交和外部刺激，從與他人互動和行動中獲得動力。
I：內向（Introversion），內向的人較注重內在的想法和感受，自處由自身想法與記憶中取得動力。
S：感覺（Sensing），感覺型人格較注重現實和具體的信息，通過眼耳鼻嗅觸味覺來認識世界。
N：直覺（Intuition），直覺型人格則較注重未來和可能性，關注於人事物背後隱含的意義及模式。
T：思考（Thinking），思考型人格較注重邏輯和分析，關注事物使用邏輯因果關係來思考。
F：情感（Feeling），情感型人格則較注重情感和人際關係，關注於人情世故，讓人與人之間和諧共處。
J：判斷（Judging）判斷型人格較注重計劃和組織，有計畫有條理，喜歡在時間框架下工作。
P：知覺（Perceiving），而知覺型人格較注重靈活和開放，喜歡以靈活、創意的方式生活（開放式結局）。

MBTI將人格分為16種類型，這16種人格類型可以用四個字母縮寫來表示，例如ISTJ（內向、感性、思考、判斷）或ENFP（外向、直覺、感受、知覺）。每種人格類型都有自己的優勢和限制。MBTI測試能夠幫助人們了解自己的人格偏好和與他人互動的方式，從而更好地適應工作、學習和生活等方面。

### 我覺得
做完測驗後，我的MBTI特質是INFP，分別為內向、直覺、情感、知覺，我覺得這個心理測驗蠻有趣的。蠻多人都覺得它比星座還要準，我自己覺得主要是因為這個測驗回答的人就是自己，所以當然比星座還準~此外，我認為這項心理測驗無法測出客觀的人格，因為它是依據受測者的想法去判斷，但受測者不可能100%的客觀，所以它測出的會是主觀認為自己的人格特質。有趣的是，之前請一個跟我很熟的朋友依據我的行為去幫我測驗，結果測出來四個字都是相反的。

**Comment by TW-Maverick** [comment]
我測完覺得真的很準，蠻符合自己的個性，就像你說的是較主觀的認定。測完之後它會跟你報告優缺點，可以讓我們更了解自己，其實是一個值得嘗試的好方法。
Ex. 它說我「不願改變」，覺得非常貼切，我會試著去解決這個問題！

**Comment by brian431** [comment]
我跟你只差一個，我是intp。身邊的人是覺得蠻像我的（？。這種基於回答問題的分類測驗有比較高的準度也是挺正常的。

**Comment by poweichen00** [comment]
我也覺得超準，我是INTJ，MBTI已經存在很久了，它的準度跟信度都應該很高。

**Comment by Miinteaa** [comment]
我是ENTP！測了好幾次都是一樣的結果，也覺得大部分ENTP的特質蠻像我自己的，就連身邊的朋友都這麼說🤣
雖然我覺得很難把世界上的所有人完全歸類成16種人格，畢竟每個人都是一個獨立的個體，個性及行事風格會受到成長環境及家庭、個人價值觀等不同的因素影響，進而發展出各式各樣的獨特性，但這個測驗也能夠讓我們更加了解自己、或是外界對於自身可能的看法。

**Comment by ALAN30359704** [comment]
我是INTJ，雖然測出來的結果是如此，但我認為我們也可以以不同的個性過出不同的生活。

**Comment by zilinng** [comment]
我也是INFP!可是我覺得跟自己沒有很像，不知道為甚麼...不過聽說不是每次測結果都會一樣，測驗結果會根據當時的狀況改變，跟一陳不變的星座比起來，我覺得可信度比較高。

**Comment by Evening218** [comment]
我是INTP~感覺似乎是有準,但我也覺得每次測出來會有點不一樣,像我就是每次測驗都變得越來越內向

**Comment by Superlan8** [comment]
我測過兩次，測過兩次出來結果有點不太一樣，我認為會導致這種原因是因為，畢竟還是心理測驗，所以年齡、時間抑或是心情等心理變化都會造成結果的不同，建議大家還是當參考就好。不要把測驗結果當成判斷別人的標準，不然會有先入為主觀念產生。

**Comment by ziihui** [comment]
我覺得mbti超準！而且我每次測都是enfp！
我覺得Mbti最準的地方是分辨e人或是I人
但不是說外向就是e人內向就是I人
而是各自獲得能量的方式是跟人群聚在一起或者獨處
原本覺得自己明明是I人，但後來觀察自己後發現真的跟人群在一起的時候都會覺得很有活力，很喜歡跟人相處～

**Comment by Elaine2899** [comment]
我印象我是ENFJ，這個性格裡提到許多我非常認同我擁有的個性，像是喜歡教導別人然後沒有意識到他人可能並沒有做好做出改變的決心，自己一股腦地想要改變別人；或是過於不切實際，常給自己施加太多壓力，認為自己應該要解決所有問題等等，MBTI說出我的這些缺點不只是讓我知道，也讓我一天天、一次次的努力改掉自己的缺點。裡面還說了不少，我質疑我我具備的優點，像是領導魅力、以及可靠性，這些是我希望我擁有並能善加使用的能力，但我並沒有自信相信自己具備這樣的長處。我非常喜歡自己測出來的這個個性ENFJ，也希望自己能朝這個個性的優點持續前進。

**Comment by alicia0725** [comment]
最近mbti 的測試有更新了！還滿多人測出不同的結果，我是ENFJ，我認為mbti 真的蠻有趣的，幫我們依照行為跟想法分成16種人，E跟I的分類等等已經很常變成大家初識會問的問題，比星座準確很多也很適合拿來聊天

**Comment by Chiuhsianglin** [comment]
有一陣子我每次測的結果都不一樣，最後一次測的時候是ISFP，很大不同的是我外向內向%變成了51/49，自己也非常的驚訝，我覺得在做這個測驗的時候也可以順便認識最近的自己，會有很大的收穫
